geth/params/defaults.go

```diff
@@ -105,4 +107,12 @@ const (
 
 	// StatusChainNetworkID is id of a test network (private chain)
 	StatusChainNetworkID = 777
+
+	// WhisperDiscv5Topic used to register and search for whisper peers using discovery v5.
+	WhisperDiscv5Topic = discv5.Topic("whisper")
+)
+
+var (
+	// WhisperDiscv5Limits declares min and max limits for peers with whisper topic.
+	WhisperDiscv5Limits = Limits{2, 2}
```

Comment:
Understand this as lower limit 2 but also upper limit 2, more are not allowed. Am I right?

Reply:
right, i think for mobile devices it doesn't make sense to have different upper and lower bounds cause we will stop discovery when max bound is reached anyway